frontend/interfaces/vulnerabilities.ts

```diff
@@ -0,0 +1,17 @@
+import PropTypes from "prop-types";
+
+export default PropTypes.shape({
+  id: PropTypes.number,
+  uid: PropTypes.number,
+  username: PropTypes.string,
+  type: PropTypes.string,
+  groupname: PropTypes.string,
+});
+
+export interface IVulnerabilities {
+  id: number;
+  uid: number;
+  username: string;
+  type: string;
+  groupname: string;
+}
```

Comment:
Looks like this was perhaps copied from users and needs to be updated for vulnerabilities?